src/main/java/at/hannibal2/skyhanni/features/dungeon/DungeonRankTabListColor.kt

```diff
@@ -34,7 +34,6 @@ object DungeonRankTabListColor {
             val sbLevel = group("sbLevel")
             val rank = groupOrNull("rank") ?: ""
             val playerName = group("playerName")
-            //val symbols = group("symbols")
```

Comment:
Instead of removing this it should just have a the proper space between the slashes and the commented code